codebase/config/sync/user.role.authenticated.yml

```diff
@@ -22,7 +24,13 @@ permissions:
   - 'access content'
   - 'access shortcuts'
   - 'cancel account'
+  - 'display eu cookie compliance popup'
   - 'opt-in or out of matomo tracking'
   - 'restful get oai_pmh'
   - 'use text format basic_html'
+  - 'view any article content'
+  - 'view any homepage content'
+  - 'view any islandora_object content'
+  - 'view any page content'
+  - 'view any study content'
```

Comment:
even with these permissions, users still need to have the correct role (manager, admin) or group membership (contributor)